spec/gameSpec.js

```diff
@@ -0,0 +1,108 @@
+describe('Game', function () {
+
+  var game;
+
+  beforeEach(function () {
+    game = new Game
+    frame = jasmine.createSpyObj('frame', {
```

Comment:
Rather than generating all your test objects here, I would get into the habit of defining helper functions 'buildFrame()', 'buildStrikeFrame()' etc and calling them in your tests. Some reasons for this:

1) It cuts down on the amount of test setup you have to do for each test
2) It's clearer in the test where your objects are coming from
3) You can extend the helper functions to allow for slightly different setups, e.g:

```
function buildFrame(firstRoll = 3, score = 7) {
  return {
    'score': score,
    'isSpare': false,
    'firstRoll': firstRoll
  }
}
```

You can then use this more easily when you want create to frame stubs with different values (